client/main.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+import React from 'react';
+import { render } from 'react-dom';
+import Http from 'axios/dist/axios';
```

Comment:
Also since I'm used to using airbnb style guide I failed to find a recommendation for this example. The closest one I found is [this](https://github.com/airbnb/javascript#naming--PascalCase-singleton), am I correct?